cmd/opm/generate/cmd.go

```diff
@@ -42,10 +43,21 @@ Dockerfile with the same name already exists, this command will fail.
 
 When specifying extra labels, note that if duplicate keys exist, only the last
 value of each duplicate key will be added to the generated Dockerfile.
+
+A separate builder and binary image can be specified. The builder image may not be "scratch".
```

Comment:
```suggestion
A separate builder and base image can be specified. The builder image may not be "scratch".
```

Reply:
While we're at it, we could also change the `dc` and `declarative config` references over to `FBC` and `file-based catalog`.